libraries/radpdfprocessing/editing/block.md

```diff
@@ -12,7 +12,7 @@ position: 0
 
 
 
-__Block__ is a utility class that is intended to arrange the elements added to it in a flow-like manner. It can be used for measuring, drawing, and splitting of __FixedContentElements__.
+The __Block__ class is intended to arrange the elements added to it in a flow-like manner. It can be used for measuring, drawing, and splitting of __FixedContentElements__.
       
 
 ## Add and Modify Content
```

Comment:
Add and Modify Content
------------
Inserting Form-XObject Elements
---------------

May we can change those headers to use -ing or not use it. The idea is to be consistent. For example:

"Adding and modifying content" and "Inserting Form-XObject Elements"
or
"Add and modify content" and "Insert Form-XObject Elements"